src/domain/space/account/account.resolver.fields.ts

```diff
@@ -137,6 +140,45 @@ export class AccountResolverFields {
     return loader.load(account.id);
   }
 
+  @ResolveField('activeSubscription', () => IAccountSubscription, {
+    nullable: true,
+    description: 'The "highest" subscription active for this Account.',
+  })
+  async activeSubscription(@Parent() account: Account) {
+    const licensingFramework =
+      await this.licensingService.getDefaultLicensingOrFail();
+
+    const today = new Date();
+    const plans = await this.licensingService.getLicensePlans(
+      licensingFramework.id
+    );
+
+    const activeSubscriptions = (
```

Comment:
You can just take the first one here and not use activeSubscriptions[0]?.subscription below